### ISSUE TITLE
64bit Upgrade

### ISSUE THREAD

**Issue by Infernio** [wrye-bash/wrye-bash#481]
*(Opening this so branches and commits have an issue to reference)*

CPython seems to be ~30% faster on 64bit, and it's high time we switched anyways.
Current blockers:
 - [x] ~~CBash seems to work, but `cint` is not passing it the right argument types (longs instead of pointers, which happens to work on 32bit)~~ - obsoleted by #530.
 - [x] `loot-api-python`. 64bit versions are available starting from version 4.0, but see #431 for the obvious problem there - cleared in 934f3527e59ac8b0f923c2f6dc6908ad902a1c46.

**Comment by Infernio** [comment]
Ref https://github.com/wrye-bash/CBash/issues/23

**Comment by Infernio** [comment]
First 64bit nightly has been built. Some experiences:
 - Memory usage is higher. Makes sense, all pointers used by python are now twice as large.
 - The performance increase I saw for PBash isn't all that consistent (on Skyrim SE I get ~30% speedup, on Skyrim LE it's only ~15%), but it's definitely there.
 - Wheels are an issue - [Gohlke's](https://www.lfd.uci.edu/~gohlke/pythonlibs/) aren't meant for automated download, so the URL changes every once in a while. I rehosted them on dev-tools for now, but this is obviously a problem that will solve itself once #460 is done.
 - We definitely want to keep the static typing in cint. Made finding and fixing 245430449b4b0d7bafe13485c9cc1260c4034b8f much easier.
